tests/helper/helper_generic.go

```diff
@@ -306,17 +311,55 @@ func CommonBeforeEach() CommonVar {
 	err := cfg.SetConfiguration(preference.ConsentTelemetrySetting, "false")
 	Expect(err).To(BeNil())
 	SetDefaultDevfileRegistryAsStaging()
+	// Ginkgo test related variables
+	commonVar.testFileName = strings.Replace(CurrentGinkgoTestDescription().FileName[strings.LastIndex(CurrentGinkgoTestDescription().FileName, "/")+1:strings.LastIndex(CurrentGinkgoTestDescription().FileName, ".")], "_", "-", -1) + ".go"
+	commonVar.testCase = CurrentGinkgoTestDescription().FullTestText
+	commonVar.testFailed = CurrentGinkgoTestDescription().Failed
+	commonVar.testDuration = CurrentGinkgoTestDescription().Duration.Seconds()
 	return commonVar
 }
 
 // CommonAfterEach is common function that cleans up after every test Spec (It)
 func CommonAfterEach(commonVar CommonVar) {
+	// Get details, including test result for each test spec and adds it to local testResults.txt file
+	var prNum string
+	var K8SorOcp string
+	var resultsRow string
+	prNum = os.Getenv("GIT_PR_NUMBER")
+	K8SorOcp = os.Getenv("KUBERNETES")
+	passedOrFailed := "PASSED"
+	if commonVar.testFailed {
+		passedOrFailed = "FAILED"
+	}
+	clusterType := "OCP"
+	if K8SorOcp == "KUBERNETES" {
+		clusterType = "KUBERNETES"
+	}
+	now := time.Now()
+	y, m, d := now.Date()
+	testDate := strconv.Itoa(y) + "-" + strconv.Itoa(int(m)) + "-" + strconv.Itoa(d)
```

Comment:
```suggestion
	testDate := time.Now().Format(time.RFC3339)
```

The format will be slightly different, something like this `2022-02-10T15:10:22+05:30` but I guess it could still work, no?